source/addComment.js

```diff
@@ -1,3 +1,13 @@
 $(document).ready(function(){
-  // add your code here
+  var commentForm = $(".comment-form");
+
+  $("#new_comment_button").on("click", function(event){
+    event.preventDefault();
+    commentBody = $("#comment-body").val()
```

Comment:
try to avoid setting global variables. be sure to put `var` in front of commentBody here